src/network/serverpackethandler.cpp

```diff
@@ -1478,12 +1479,12 @@ void Server::handleCommand_Interact(NetworkPacket* pkt)
 			// If the object is a player and its HP changed
 			if (src_original_hp != pointed_object->getHP() &&
 					pointed_object->getType() == ACTIVEOBJECT_TYPE_PLAYER) {
-				SendPlayerHPOrDie((PlayerSAO *)pointed_object);
+				SendPlayerHPOrDie((PlayerSAO *)pointed_object, PlayerHPChangeReason(PlayerHPChangeReason::PLAYER_PUNCH, playersao));
```

Comment:
if i'm correct C++11 notation {PlayerHPChangeReason::PLAYER_PUNCH, playersao} can be used instead

Reply:
this doesn't scale when adding more parameters (which I may do)